src/components/ui/card.tsx

```diff
@@ -0,0 +1,144 @@
+import { forwardRef, HTMLAttributes } from 'react';
+
+import { CalendarDays, CircleCheck } from 'lucide-react';
+
+import { cn } from '@/lib/utils';
+
+interface CardHeaderProps extends HTMLAttributes<HTMLDivElement> {
+  startDate: string;
+  endDate: string;
+}
+
+interface CardContentProps extends HTMLAttributes<HTMLDivElement> {
+  category: string;
+  tenant: string;
+  isComplete: boolean;
+  isLast?: boolean;
+}
+
+/**
+ * @example
+ * ```ts
+ * return (
+ * <Card>
+ *  <CardHeader startDate={startDate} endDate={endDate} />
```

Comment:
Could you update this example `CardHeader` to take `title` props?

Reply:
Thank you for re-review! I fixed the TSDoc!